docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,146 @@
+
+![edit-user](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-3-edit-user.png)
+
+### 4) Merchant and operator: Create and update the product and offers
+
+Operator: Decide who the owner of the product data is:
+* If the operator is the owner and manages the product data, other merchants can create offers with specific prices and available quantities on top of existing products.
+* If merchants can create products, the operator needs to make sure there are no duplicates in the Marketplace and regularly check the product data quality.
+
+In the Merchant Portal, merchants can create and update products and offers individually. Alternatively, the operator can help the merchant set up the process for automatically importing products and offers from a CSV template or establish the data exchange process from the merchant's PIM or ERP system.
+
+{% info_block infoBox %}
+
+Based on the product volume, we recommend the following:
+- If the number of products to be created or edited is small, merchants can use the interface in Merchant Portal.
+- For large volumes, use a file and data importers or integrate with the merchant PIM or ERP system through Spryker Middleware.
+
+{% endinfo_block %}
+
+The operator and merchants need to align on categories, attributes, and values that can be used in the importing process. Mapping needs to be done by merchants or through Spryker Middleware.
+
+![concrete-products](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-4-concrete-products.png)
+
+![create-offer](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-4-create-offer.png)
+
+
+### 5) Operator: Set up the payment process
+
+The following steps are taken by the operator:
+1. Decide what kind of payment process to implement—for example, whether to use a PSP or not. 
+2. Choose a revenue model to apply in the marketplace—for example, transaction-based, subscription-based, or listing-based—and whether a PSP can cover it.
+
+The following example illustrates the process when using a PSP to cover the marketplace payments completely:
+1. The merchant registers on the PSP portal and provides relevant company and representative information.
+2. The PSP checks the merchant during the compliance and KYC process.
+3. PSP approves the merchant and starts technical onboarding.
+4. The PSP calculates transactional fees and commissions agreed upon and makes payments to merchants.
+5. The merchant receives access to the PSP Back Office, where data regarding commissions and payments can be checked.
+
+{% info_block infoBox %}
+
+For more information about the payment setup and available options, ask your Spryker contact.
+
+{% endinfo_block %}
```

Comment:
would be good to give a link to doc on how to create products in the Merchant portal

Reply:
Oh, sorry, my comment actually was for section that starts with line 67 here. And yes, we have the relevant doc in the merchant portal, here: https://docs.spryker.com/docs/marketplace/user/merchant-portal-user-guides/202212.0/products/abstract-products/creating-marketplace-abstract-product.html Would be good to create a landing page for the whole "Products" section though